src/main/java/redis/clients/jedis/params/XReadGroupParams.java

```diff
@@ -0,0 +1,31 @@
+package redis.clients.jedis.params;
+
+public class XReadGroupParams extends Params {
+
+  private static final String COUNT = "COUNT";
+  private static final String BLOCK = "BLOCK";
+  private static final String NOACK = "NOACK";
+
+  public static XReadGroupParams xReadGroupParams() {
```

Comment:
@sazzad16 I would have consider a different name for `xReadGroupParams()` perhaps `create()`

Reply:
Seems that static method is not really necessary. Because user can also new Object by `none args Constructor`. Maybe we can also keep `none args Constructor` private?

Reply:
@gkorland It is there for method chaining. Any _good enough_ name is okay to me.

@dengliming Empty constructor is still an option to the users if they want to use. I don't to hide it until it is hurting the development process.

Reply:
@gkorland We are carrying this pattern probably since Jedis 2.10.0. Each of the Params classes have a static method with name at this pattern. Some of already merged PR's for 3.6.0 include this pattern. If we want replace this pattern with something like `create()`, we'll always have time to do that. IMHO, we shouldn't keep this PR on hold just for this matter if we agree on the implementation of this PR.

Reply:
> Each of the Params classes have a static method with name at this pattern.

If that is the common pattern then lest leave it like that